sympy/physics/control/lti.py

```diff
@@ -0,0 +1,655 @@
+from sympy import Basic, Mul, degree, Symbol, expand, cancel, Expr, exp
+from sympy.core.evalf import EvalfMixin
+from sympy.core.numbers import Integer
+from sympy.core.sympify import sympify, _sympify
+
+__all__ = ['TransferFunction', 'Series', 'Parallel', 'Feedback']
+
+
+class TransferFunction(Basic, EvalfMixin):
+    """TransferFunction(num, den, var)
+
+    A class for representing continuous transfer functions. This class is used to represent
+    LTI (Linear, time-invariant) systems in transfer function form. The arguments
+    are ``num``, ``den``, and ``var``, where ``num`` and ``den`` are numerator and
+    denominator polynomials of the ``TransferFunction`` respectively, and the third argument is
+    a variable used to anchor these polynomials of the transfer function. ``num`` and ``den`` can
+    be either sympy expressions (with no time delay terms) or numbers, whereas ``var``
+    has to be a Symbol.
+
+    Parameters
+    ==========
+
+    num : Expr, Number
+        The numerator polynomial of the transfer function.
+    den : Expr, Number
+        The denominator polynomial of the transfer function.
+    var : Symbol
+        Variable used to anchor the polynomials of the transfer function.
```

Comment:
I am not a fan of the term "anchor". I'm not sure what that means mathematically. This is the complex variable of the Laplace Transform. We should call it that.